configure

```diff
@@ -165,8 +167,9 @@ def main():
         if config.target.sdk == OSType.MacOSX and config.system_root is None and Target(Target.default()).sdk == OSType.MacOSX:
             import subprocess
             config.system_root = Path.path(subprocess.Popen(['xcrun', '--show-sdk-path'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE).communicate()[0])
-            swift_path = Path.path(subprocess.Popen(['xcrun', '--find', 'swift'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE).communicate()[0]).parent().parent()
-            config.swift_sdk = swift_path.absolute()
+            if config.swift_sdk is None:
+                swift_path = Path.path(subprocess.Popen(['xcrun', '--find', 'swift'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE).communicate()[0]).parent().parent()
+                config.swift_sdk = swift_path.absolute()
```

Comment:
I like these changes; it seems like a pretty sensible isolated set of changes. Can we extract just this into a commit?